app/src/main/kotlin/com/wire/android/ui/home/HomeStateHolder.kt

```diff
@@ -78,13 +75,13 @@ class HomeStateHolder(
 
     fun openBottomSheet() {
         coroutineScope.launch {
-            if (!bottomSheetState.isVisible) bottomSheetState.animateTo(ModalBottomSheetValue.Expanded)
+            if (!bottomSheetState.isVisible) bottomSheetState.show()
         }
     }
 
     fun closeBottomSheet() {
         coroutineScope.launch {
-            if (bottomSheetState.isVisible) bottomSheetState.animateTo(ModalBottomSheetValue.Hidden)
+            if (bottomSheetState.isVisible) bottomSheetState.hide()
```

Comment:
No animation ?

Reply:
`animateTo` is `internal` in material3 but we have `hide`, `show`, `expand` and `partialExpand` functions, all of which execute `animateTo` under the hood 😃